pages/domains/domains_index.md

```diff
@@ -1,8 +1,8 @@
 ---
-title: Domains
+title: .Gov domains
 permalink: /domains/
 layout: layouts/landing
-excerpt: Learn about .gov domains
+excerpt: Find out if you're eligible for a .gov domain. Learn how to get a .gov domain. Get tips for moving to .gov from another top-level domain.
```

Comment:
Is there a benefit to keeping the excerpts short? I feel like the "Learn how to get a .gov domain." sentence encompasses the other sentences used here, if it beneficial to keep it short.